modules/twinklewarn.js

```diff
@@ -1012,8 +1012,8 @@ Twinkle.warn.messages = {
 
 	singlewarn: {
 		"uw-3rr": {
-			label: "Violating the three-revert rule; see also uw-ew",
-			summary: "Warning: Violating the three-revert rule"
+			label: "Three-revert rule; see also uw-ew",
```

Comment:
The label is only seen by Twinkle users, so it doesn't have to change. If you think it should be changed, I'd prefer to see something slightly more descriptive than just "three-revert rule". Could two or three more words be added?